bemani/utils/read.py

```diff
@@ -883,15 +884,15 @@ def available_charts(mask: int) -> Tuple[bool, bool, bool, bool, bool, bool]:
                     mask & 0x4000000 > 0,  # Battle hyper chart bit
                 )
         elif self.version == VersionConstants.POPN_MUSIC_USANEKO:
-            # Based on M39:J:A:A:2018101500
```

Comment:
Why earlier?

Reply:
This is the latest date code that I could personally find from any source I have. You probably have better sources than me lol